internal/runner/cypress_test.go

```diff
@@ -0,0 +1,205 @@
+package runner
+
+import (
+	"errors"
+	"os/exec"
+	"syscall"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kballard/go-shellquote"
+)
+
+func TestCypressRun(t *testing.T) {
+	mockCwd(t, "./testdata/cypress")
```

Comment:
[Note for something defined outside this PR] `mockCwd` _actually really for real_ changes the working directory for the duration of the test, so it's not a "mock"!

Reply:
Oh thanks! I didn't think deeply about this. Maybe `changeWorkingDirectory`  more accurate?